cli/internal/prune/prune.go

```diff
@@ -191,7 +193,34 @@ func (p *prune) prune(opts *turbostate.PrunePayload) error {
 	}
 
 	if fs.FileExists("turbo.json") {
-		if err := fs.CopyFile(&fs.LstatCachedFile{Path: p.base.RepoRoot.UntypedJoin("turbo.json")}, fullDir.UntypedJoin("turbo.json").ToStringDuringMigration()); err != nil {
+		turboJSON, err := fs.ReadTurboConfig(".", rootPackageJSON)
```

Comment:
We should drop the `fs.FileExists` condition:
1. It has implicit cwd behavior.
2. We can just try to read, handle any error, and reduce syscalls as a consequence. (This will also make sure we have a viable error API for `ReadTurboConfig`.)

(If you want to pick off the same thing for `.gitignore` above that'd also be good.)

Also, 
```suggestion
		turboJSON, err := fs.ReadTurboConfig(p.base.RepoRoot, rootPackageJSON)
```

"." is accidentally abusing the fact that primitives don't get typechecked on the way in to the `ReadTurboConfig` call.